src/hdf5_time_series_storage.jl

```diff
@@ -480,7 +480,7 @@ function deserialize_time_series(
         data = SortedDict{Dates.DateTime, Array}()
         start_time = attributes["start_time"]
         if length(columns) == 1
-            data[start_time] = path["data"][columns, rows, 1:total_percentiles]
+            data[start_time] = path["data"][1, rows, 1:total_percentiles]
```

Comment:
Why did you need to do this?  Don't we need to be able to read from an offset other than one? The code in the else block uses `columns`.

Reply:
We need to pass a single value. columns here is a range, it should then be `end(columns)` if `columns` is a single one.